web/docs/data-model/prisma-file.md

```diff
@@ -0,0 +1,200 @@
+├── tsconfig.json
+└── vite.config.ts
+```
+
+ Wasp uses the `schema.prisma` file to understand your app's data model and generate the necessary code to interact with the database.
+
+## Wasp file and Prisma schema file
+
+Let's see how Wasp and Prisma files work together to define your application.
+
+Here's an example `schema.prisma` file where we defined some database options and two models (User and Task) with a one-to-many relationship:
+
+```prisma title="schema.prisma"
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+model User {
+  id      Int        @id @default(autoincrement())
+  tasks   Task[]
+}
+
+model Task {
+  id          Int        @id @default(autoincrement())
+  description String
+  isDone      Boolean    @default(false)
+  user        User       @relation(fields: [userId], references: [id])
+  userId      Int
+}
+```
+
+Wasp reads this `schema.prisma` file and extracts the info about your database models and database config:
+
+<ImgWithCaption alt="Relationship between Wasp file and Prisma file" source="img/data-model/prisma_in_wasp.png" caption="Relationship between Wasp file and Prisma file"/>
+
+Wasp uses the `datasource` block to know which database you want to use (PostgreSQL in this case) and some other options.
+
+Wasp uses the `generator` block to generate the Prisma Client code that you can use in your application to interact with the database.
+
+Finally, Prisma models represent Wasp Entities which can be then used in the `main.wasp` file:
+
+```wasp title="main.wasp"
+app myApp {
+  wasp: {
+    version: "^0.14.0"
+  },
+  title: "My App",
+}
+
+...
+
+// Using Wasp Entities in the Wasp file
+
+query getTasks {
+  fn: import { getTasks } from "@src/queries",
+  // highlight-next-line
+  entities: [Task]
+}
+
+job myJob {
+  executor: PgBoss,
+  perform: {
+    fn: import { foo } from "@src/workers/bar"
+  },
+  // highlight-next-line
+  entities: [Task],
+}
+
+api fooBar {
+  fn: import { fooBar } from "@src/apis",
+  // highlight-next-line
+  entities: [Task],
+  httpRoute: (GET, "/foo/bar/:email")
+}
+
+```
+
+In the implementation of the `getTasks` query, `Task` is a Wasp Entity that corresponds to the `Task` model defined in the `schema.prisma` file. 
+
+The same goes for the `myJob` job and `fooBar` API, where `Task` is used as an Entity.
+
+## Wasp-specific Prisma configuration
+
+Wasp mostly lets you use the Prisma schema file as you would in any other JS/TS project. However, there are some Wasp-specific rules you need to follow.
+
+### The `datasource` block
+
+```prisma title="schema.prisma"
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+```
+
+Wasp will take the `datasource` you write and use it as-is.
+
+There are some rules you need to follow:
+- You can only use `"postgresql"` or `"sqlite"` as the `provider` because Wasp only supports PostgreSQL and SQLite databases. 
```

Comment:
```suggestion
- You can only use `"postgresql"` or `"sqlite"` as the `provider` because Wasp only supports PostgreSQL and SQLite databases for now.
```